x-pack/plugins/security_solution/public/actions/register.ts

```diff
@@ -57,37 +59,36 @@ const registerCellActions = (
     toggleColumn: createToggleColumnCellActionFactory({ store }),
   };
 
-  registerCellActionsTrigger(uiActions, SecurityCellActionsTrigger.DEFAULT, cellActions, [
-    'filterIn',
-    'filterOut',
-    'addToTimeline',
-    'showTopN',
-    'copyToClipboard',
-  ]);
+  registerCellActionsTrigger(
+    uiActions,
+    SecurityCellActionsTrigger.DEFAULT,
+    cellActions,
+    ['filterIn', 'filterOut', 'addToTimeline', 'showTopN', 'copyToClipboard'],
+    services
+  );
 
-  registerCellActionsTrigger(uiActions, SecurityCellActionsTrigger.DETAILS_FLYOUT, cellActions, [
-    'filterIn',
-    'filterOut',
-    'addToTimeline',
-    'toggleColumn',
-    'showTopN',
-    'copyToClipboard',
-  ]);
+  registerCellActionsTrigger(
+    uiActions,
+    SecurityCellActionsTrigger.DETAILS_FLYOUT,
+    cellActions,
+    ['filterIn', 'filterOut', 'addToTimeline', 'toggleColumn', 'showTopN', 'copyToClipboard'],
+    services
+  );
```

Comment:
@machadoum Maybe this function is starting to have too many parameters 🤔  
What do you think about turning it into one object param, and passing the `cellActions` factories explicitly? Probably it will be easier to understand:
```
registerCellActionsTrigger({
    uiActions,
    services,
    triggerId: SecurityCellActionsTrigger.DETAILS_FLYOUT,
    cellActions: [
        cellActions.filterIn,
        cellActions.filterOut,
        cellActions.addToTimeline,
        cellActions.toggleColumn,
        cellActions.showTopN,
        cellActions.copyToClipboard,
    ],
  });
```